CONTRIBUTING.md

```diff
@@ -271,6 +271,9 @@ Example use cases:
 
 If a PR is marked "don't merge" and it becomes ready (i.e. you did the necessary fixups and rebased on master), please remember to remove the tag again.
 
+When your PR is ready to be merged (you have the required number of reviews and you addressed all the review comments), you don't need to wait for the CI to finish.
+You can add the "MergeWhenReady" tag, and Github will automatically merge the pull request once the CI has finished and is passing.
```

Comment:
Actually, my idea was to remove the use of the "don't merge" tag completely, and instead rely on the MergeWhenReady tag to mark it as ready to merge. So it's a bit more invasive change to our workflow.